io/plugins/eu.esdihumboldt.hale.io.topojson.test/src/eu/esdihumboldt/hale/io/topojson/test/TopoJsonInstanceWriterTest.groovy

```diff
@@ -56,6 +57,7 @@ class TopoJsonInstanceWriterTest {
 		SimpleType {
 			id(String)
 			name(String)
+			label(String)
```

Comment:
Having an additional unit test to test the case with the null date would be great.